homeassistant/components/camera/synology.py

```diff
@@ -164,10 +169,12 @@ def get_session_id(hass, username, password, login_url, valid_cert):
     }
     try:
         with async_timeout.timeout(TIMEOUT, loop=hass.loop):
-            auth_req = yield from hass.websession.get(
+            auth_req = yield from aiohttp.get(

```

Comment:
But then I need a new ClientSession every time the cookie changes.  pvizeli keeps linking me to the deprecated  aiohttp.get and telling me it accepts cookies.  ClientSession.get doesn't accept cookies, it has to be set in the connector.  I seem to be getting conflicting direction here.  Do I use aiohttp.get and pass the cookie, or create a new ClientSession for every connection that needs a different cookie?